src/utils/spawner.js

```diff
@@ -41,71 +36,67 @@ var spawner = {
-                bodyPieces.push(WORK);
-                cost += body.WORK.cost;
-            }
 
-            var code = spawner.spawnCreep(bodyPieces, createName(roles.HARVEST), {memory: {role: roles.HARVEST, source: targetSource}});
-            console.log('Spawning new harvester: ' + code);
-            return;
+        if(harvesters.length < targetHarvesters) {
+            console.log(`Harv: ${harvesters.length}/${targetHarvesters} Haul: ${haulers.length}/${targetHaulers} Buil: ${builders.length}/${targetBuilders} Upgr: ${upgraders.length}/${targetUpgraders}`);
+            spawnHarvester();
         }
         
-        if(upgraders.length < targetUpgraders) {
-            bodyPieces = [WORK, CARRY, MOVE];
-            var code = spawner.spawnCreep(bodyPieces, createName(roles.UPGRADE), {memory: {role: roles.UPGRADE}});
-            console.log('Spawning new upgrader: ' + code);
-            return;
+        else if(upgraders.length < targetUpgraders) {
+            console.log(`Harv: ${harvesters.length}/${targetHarvesters} Haul: ${haulers.length}/${targetHaulers} Buil: ${builders.length}/${targetBuilders} Upgr: ${upgraders.length}/${targetUpgraders}`);
+            spawnUpgrader();
         }
 
-        if(builders.length < targetBuilders) {
-            var bodyPieces = [MOVE, MOVE, WORK];
-            var cost = body.MOVE.cost * 2 + body.WORK.cost;
-            while(cost + body.CARRY.cost <= availableEnergy){
-                bodyPieces.push(CARRY);
-                cost += body.CARRY.cost;
-            }
-            var code = spawner.spawnCreep(bodyPieces, createName(roles.BUILD), {memory: {role: roles.BUILD}});
-            console.log('Spawning new builder: ' + code);
-            return;
+        else if(builders.length < targetBuilders) {
+            console.log(`Harv: ${harvesters.length}/${targetHarvesters} Haul: ${haulers.length}/${targetHaulers} Buil: ${builders.length}/${targetBuilders} Upgr: ${upgraders.length}/${targetUpgraders}`);
+            spawnBuilder();
         }
 
-        if(haulers.length < targetHaulers) {
-            var bodyPieces = [MOVE, CARRY];
-            var cost = body.MOVE.cost + body.CARRY.cost;
-            while(cost + body.WORK.cost + body.CARRY.cost <= availableEnergy){
-                bodyPieces.push(WORK);
-                bodyPieces.push(CARRY);
-                cost += body.WORK.cost + body.CARRY.cost;
-            }
-            var code = spawner.spawnCreep(bodyPieces, createName(roles.HAUL), {memory: {role: roles.HAUL}});
-            console.log('Spawning new hauler: ' + code);
-            return;
+        else if(haulers.length < targetHaulers) {
+            console.log(`Harv: ${harvesters.length}/${targetHarvesters} Haul: ${haulers.length}/${targetHaulers} Buil: ${builders.length}/${targetBuilders} Upgr: ${upgraders.length}/${targetUpgraders}`);
+            spawnHauler();
+        }
+
+        else {
+            //targetHarvesters += 1;
```

Comment:
Could you remove this commented line? Presumably, it was useful in development but we don't want it as part of the permanent codebase.